shared/chat/conversation/messages/message-popup/payment/index.js

```diff
@@ -0,0 +1,123 @@
+// @flow
+import * as React from 'react'
+import {toUpper, upperFirst} from 'lodash-es'
+import * as Kb from '../../../../../common-adapters'
+import * as S from '../../../../../styles'
+import type {Position} from '../../../../../common-adapters/relative-popup-hoc'
+
+type HeaderProps = {|
+  amountNominal: string,
+  balanceChange: string, // may be empty
```

Comment:
Is there a reason why you prefer passing empty strings vs having them be optional properties?

Reply:
This'll be connected during integration, so it's likely that the props will always be supplied. For them not to be something like this would have to happen in `mergeProps`:
```
return {
  ...(balanceChange ? {balanceChange} : {}
  <etc>
}
```
which I think is messy. Another benefit is that this way the prop can only have one falsy value rather than two.